src/views/GovernanceTaskList/PresentationDeckUpload/index.test.tsx

```diff
@@ -21,16 +21,16 @@ describe('UploadForm page', () => {
     );
 
     expect(
-      findByText(
+      screen.getAllByText(
         i18next.t<string>(
-          'grbReview:presentationLinks.presentationDeckUpload:dontUploadAdmin'
-        )[0]
-      )
+          'grbReview:presentationLinks.presentationUpload:dontUploadAdmin'
```

Comment:
Thanks for updating these tests! I didn't realize you could use a colon instead of a period in the translation key (other than after the namespace).